broker/core/express-commons/src/middleware/index.js

```diff
@@ -35,6 +36,35 @@ exports.basicAuth = function (username, password) {
   };
 };
 
+exports.tlsAuth = function () {
+  return function (req, res, next) {
+    const clientCertificate = req.headers['ssl-client-cert'];
+    if (!clientCertificate) {
+      logger.error('clientCertificate not found in the request headers: ', req.headers);
+      next(new Unauthorized());
```

Comment:
```suggestion
      next(new Unauthorized('Client certificate not found in the request headers'));
```
I feel it will be more informative to the user if we pass the cause of the error.  This message will be part of the response. The default message is `'The request requires user authentication'`.